<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="3" hostname="LAPTOP-DO61B2MO" failures="2" tests="2" name="Default test(failed)(failed)" time="0.035" errors="0" timestamp="2022-03-22T16:30:58 IST">
  <testcase classname="Practice.assrt" name="TEST4" time="0.006">
    <failure type="java.lang.AssertionError" message="expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at Practice.assrt.TEST4(assrt.java:31)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- TEST4 -->
  <testcase classname="Practice.assrt" name="TEST3" time="0.0">
    <failure type="java.lang.AssertionError" message="expected [20] but found [10]">
      <![CDATA[java.lang.AssertionError: expected [20] but found [10]
at Practice.assrt.TEST3(assrt.java:25)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- TEST3 -->
  <testcase classname="Practice.assrt" name="TEST5" time="0.0">
    <ignored/>
  </testcase> <!-- TEST5 -->
  <testcase classname="Practice.assrt" name="TEST2" time="0.0">
    <ignored/>
  </testcase> <!-- TEST2 -->
  <testcase classname="Practice.assrt" name="TEST1" time="0.0">
    <ignored/>
  </testcase> <!-- TEST1 -->
</testsuite> <!-- Default test(failed)(failed) -->
